`yarn start-dev` fails in my local machine due to too many open files to watch

When running starting the server in `dev` mode, I get the following error:
> `yarn start-dev`

<details>
  <summary>Click to expand!</summary>
  
  ```bash
    LOCAL_URL="http://localhost:4200" LIMIT_CONTENT_ARCHIVE=true ember server
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
node-resolve: setting options.module is deprecated, please override options.mainFields instead
node-resolve: setting options.jsnext is deprecated, please override options.mainFields instead
node-resolve: setting options.main is deprecated, please override options.mainFields instead
EMFILE: too many open files, watch
  ```
</details>

---

When installing dependencies I get some errors too which I'm not sure if they are related:

> `yarn --pure-lockfile`
<details>
  <summary>Click to expand!</summary>
  
  ```bash
    yarn install v1.22.4
[1/5] 🔍  Validating package.json...
[2/5] 🔍  Resolving packages...
[3/5] 🚚  Fetching packages...
[4/5] 🔗  Linking dependencies...
warning "@css-blocks/ember-cli > @glimmer/application-pipeline > ember-build-utilities@0.4.0" has unmet peer dependency "@glimmer/compiler@> 0.26.0".
warning " > @glimmer/inline-precompile@1.0.1" has unmet peer dependency "@glimmer/compiler@> 0.23.0 || > 0.24.0-alpha.1 || > 0.25.0-alpha.1".
warning "@glimmer/inline-precompile > babel-plugin-glimmer-inline-precompile@1.2.0" has unmet peer dependency "@glimmer/compiler@> 0.23.0 || > 0.24.0-alpha.1 || > 0.25.0-alpha.1".
warning "ember-cli-tslint > broccoli-tslinter@3.0.1" has incorrect peer dependency "typescript@^2.1.0".
warning " > rollup-plugin-uglify@6.0.4" has incorrect peer dependency "rollup@>=0.66.0 <2".
warning " > tslint-plugin-prettier@2.3.0" has unmet peer dependency "tslint@^5.0.0 || ^6.0.0".
warning "fsevents@1.2.6" is missing a bundled dependency "node-pre-gyp". This should be reported to the package maintainer.
[5/5] 🔨  Building fresh packages...
[1/4] ⠂ puppeteer
[2/4] ⠂ puppeteer
[3/4] ⠂ puppeteer
warning Error running install script for optional dependency: "/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents: Command failed.
Exit code: 1
Command: node install
Arguments: 
Directory: /Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents
Output:
node-pre-gyp info it worked if it ends with ok
node-pre-gyp info using node-pre-gyp@0.10.3
node-pre-gyp info using node@12.22.3 | darwin | x64
node-pre-gyp WARN Using request for node-pre-gyp https download 
node-pre-gyp info check checked for \"/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents/lib/binding/Release/node-v72-darwin-x64/fse.node\" (not found)
node-pre-gyp http GET https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.6/fse-v1.2.6-node-v72-darwin-x64.tar.gz
node-pre-gyp http 404 https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.6/fse-v1.2.6-node-v72-darwin-x64.tar.gz
node-pre-gyp WARN Tried to download(404): https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.6/fse-v1.2.6-node-v72-darwin-x64.tar.gz 
node-pre-gyp WARN Pre-built binaries not found for fsevents@1.2.6 and node@12.22.3 (node-v72 ABI, unknown) (falling back to source compile with node-gyp) 
node-pre-gyp http 404 status code downloading tarball https://fsevents-binaries.s3-us-west-2.amazonaws.com/v1.2.6/fse-v1.2.6-node-v72-darwin-x64.tar.gz 
node-pre-gyp WARN Pre-built binaries not installable for fsevents@1.2.6 and node@12.22.3 (node-v72 ABI, unknown) (falling back to source compile with node-gyp) 
node-pre-gyp WARN Hit error Connection closed while downloading tarball file 
gypgyp info  it worked if it ends with ok
info it worked if it ends with ok
gypgyp  infoinfo  usingusing node-gyp@5.1.0
 node-gyp@5.1.0
gypgyp  infoinfo  usingusing node@12.22.3 | darwin | x64
 node@12.22.3 | darwin | x64
gyp info gypok 
 info ok 
gyp info it worked if it ends with ok
gyp info using node-gyp@5.1.0
gyp info using node@12.22.3 | darwin | x64
gyp info it worked if it ends with ok
gyp info using node-gyp@5.1.0
gyp info using node@12.22.3 | darwin | x64
gypgyp info  find Pythoninfo using Python version 3.9.12 found at \"/usr/local/opt/python@3.9/bin/python3.9\"
 find Python using Python version 3.9.12 found at \"/usr/local/opt/python@3.9/bin/python3.9\"
gypgyp info spawn  /usr/local/opt/python@3.9/bin/python3.9
info spawn /usr/local/opt/python@3.9/bin/python3.9
gyp info gypspawn args  [
infogyp  infospawn args spawn args   '/Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
 [
gypgyp  infoinfo  spawn argsspawn args   '/Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
   'binding.gyp',
gypgyp  info infospawn args    'binding.gyp',
spawn argsgyp   '-f',
 gypinfo  infospawn args    '-f',
spawn argsgyp   'make',
 gypinfo  infospawn args    'make',
spawn argsgyp   '-I',
 gypinfo  infospawn args    '-I',
spawn argsgyp   '/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents/build/config.gypi',
 gypinfo  infospawn args    '/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents/build/config.gypi',
spawn argsgyp   '-I',
 gypinfo  infospawn args    '-I',
spawn argsgyp   '/Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
 gypinfo  infospawn args    '/Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
spawn argsgyp   '-I',
 gypinfo  infospawn args    '-I',
spawn argsgyp   '/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/common.gypi',
 gypinfo  infospawn args    '/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/common.gypi',
spawn argsgyp   '-Dlibrary=shared_library',
 gypinfo  infospawn args    '-Dlibrary=shared_library',
spawn argsgyp   '-Dvisibility=default',
 gypinfo  spawn args   '-Dvisibility=default',
gyp info spawn argsinfo   '-Dnode_root_dir=/Users/XXX/Library/Caches/node-gyp/12.22.3',
gyp  spawn args   '-Dnode_root_dir=/Users/XXX/Library/Caches/node-gyp/12.22.3',
gyp infoinfo  spawn argsspawn args   '-Dnode_gyp_dir=/Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp',
   '-Dnode_gyp_dir=/Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp',
gypgyp  infoinfo  spawn args   '-Dnode_lib_file=/Users/XXX/Library/Caches/node-gyp/12.22.3/<(target_arch)/node.lib',
gyp spawn argsinfo    '-Dnode_lib_file=/Users/XXX/Library/Caches/node-gyp/12.22.3/<(target_arch)/node.lib',
spawn argsgyp   '-Dmodule_root_dir=/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp  info spawn argsinfo   '--generator-output',
 spawn args   '-Dmodule_root_dir=/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents',
gyp info spawn args   '-Dnode_engine=v8',
gyp info spawn args   '--depth=.',
gyp info spawn args   '--no-parallel',
gyp info spawn args   '--generator-output',
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gyp info spawn args   'build',
gyp info spawn args   '-Goutput_dir=.'
gyp info spawn args ]
gypgyp info ok  
info ok 
gypgyp info it worked if it ends with ok
 info it worked if it ends with ok
gyp infogyp using  node-gyp@5.1.0
info using node-gyp@5.1.0
gyp info using node@12.22.3 | darwin | x64
gyp info using node@12.22.3 | darwin | x64
gypgyp info spawn  make
info spawn make
gyp info gypspawn args info [ 'BUILDTYPE=Release', '-C', 'build' ]
 spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
  SOLINK_MODULE(target) Release/.node
  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
  CXX(target) Release/obj.target/fse/fsevents.o
In file included from ../fsevents.cc:6:
In file included from ../../../../nan/nan.h:222:
In file included from In file included from ../fsevents.cc:6:
In file included from ../../../../nan/nan.h:222:
In file included from ../../../../nan/nan_converters.h:67../../../../nan/nan_converters.h::
67:
../../../../nan/nan_converters_43_inl.h:22:1: warning: 'ToBoolean' is deprecated: ToBoolean can never throw. Use Local version. [-Wdeprecated-declarations]../../../../nan/nan_converters_43_inl.h:22:1
: warning: 'ToBoolean' is deprecated: ToBoolean can never throw. Use Local version. [-Wdeprecated-declarations]
X(Boolean)
^
../../../../nan/nan_converters_43_inl.h:18:12: note: expanded from macro 'X'
X(Boolean)
^      val->To ## TYPE(isolate->GetCurrentContext())                            \\\n
           ^../../../../nan/nan_converters_43_inl.h
:18:12:<scratch space> :note: 161expanded from macro 'X':
1: note: expanded from here
ToBoolean
^
      val->To ## TYPE(isolate->GetCurrentContext())                            \\\n           ^
<scratch space>:161:1: note: expanded from here
ToBoolean
^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:2663:3: note: 'ToBoolean' has been explicitly marked deprecated here
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:2663:3: note: 'ToBoolean' has been explicitly marked deprecated here
  V8_DEPRECATED(\"ToBoolean can never throw. Use Local version.\",  V8_DEPRECATED(\"ToBoolean can never throw. Use Local version.\",
  ^
  ^

/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:2929: : note: note: expanded from macro 'V8_DEPRECATED'
expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:6:
In file included from ../../../../nan/nan.h:222:
In file included from ../../../../nan/nan_converters.h:67:
../../../../nan/nan_converters_43_inl.h:40:1: warning: 'BooleanValue' is deprecated: BooleanValue can never throw. Use Isolate version. [-Wdeprecated-declarations]
X(bool, Boolean)
^
../../../../nan/nan_converters_43_inl.h:37:15: note: expanded from macro 'X'
In file included from ../fsevents.cc:6:
In file included from ../../../../nan/nan.h:222:
In file included from ../../../../nan/nan_converters.h:  return val->NAME ## Value(isolate->GetCurrentContext());                     \\67
:
              ^../../../../nan/nan_converters_43_inl.h
:40:1:<scratch space> :168:1:warning:  note: 'BooleanValue' is deprecated: BooleanValue can never throw. Use Isolate version. [-Wdeprecated-declarations]expanded from here

X(bool, Boolean)
^BooleanValue

^
../../../../nan/nan_converters_43_inl.h:37:/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h15::2701 :note: 3expanded from macro 'X':
 note: 'BooleanValue' has been explicitly marked deprecated here
  V8_DEPRECATED(\"BooleanValue can never throw. Use Isolate version.\",
  return val->NAME ## Value(isolate->GetCurrentContext());                     \\\n  ^
              ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:<scratch space>:29:168:1: note: expanded from macro 'V8_DEPRECATED'
 note: expanded from here
  declarator __attribute__((deprecated(message)))
                            ^
BooleanValue
^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:2701:3: note: 'BooleanValue' has been explicitly marked deprecated here
  V8_DEPRECATED(\"BooleanValue can never throw. Use Isolate version.\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from In file included from ../fsevents.cc:6:
In file included from ../../../../nan/nan.h:223:
In file included from ../../../../nan/nan_new.h:189:
../../../../nan/nan_implementation_12_inl.h:356:37: error: too few arguments to function call, expected 2, have 1
../fsevents.cc:6:
In file included from ../../../../nan/nan.h:223:
In file included from ../../../../nan/nan_new.h  return v8::StringObject::New(value).As<v8::StringObject>();
         ~~~~~~~~~~~~~~~~~~~~~      ^:189:
../../../../nan/nan_implementation_12_inl.h
:356:37:/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h :error: 5531too few arguments to function call, expected 2, have 1:
23: note: 'New' declared here
  return v8::StringObject::New(value).As<v8::StringObject>();
  static Local<Value> New(Isolate* isolate, Local<String> value);
         ~~~~~~~~~~~~~~~~~~~~~      ^
                      ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:5531:23: note: 'New' declared here
  static Local<Value> New(Isolate* isolate, Local<String> value);
                      ^
In file included from ../fsevents.cc:6:
In file included from ../../../../nan/nan.h:2722:
../../../../nan/nan_object_wrap.h:24:25:In file included from ../fsevents.cc:6:
In file included from ../../../../nan/nan.h:2722 error: no member named 'IsNearDeath' in 'Nan::Persistent<v8::Object>':

../../../../nan/nan_object_wrap.h:24:25: error: no member named 'IsNearDeath' in 'Nan::Persistent<v8::Object>'
    assert(persistent().IsNearDeath());
           ~~~~~~~~~~~~ ^
    assert(persistent().IsNearDeath());
           ~~~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/assert.h:99:25/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/assert.h:: note: 99expanded from macro 'assert':
25: note: expanded from macro 'assert'
    (__builtin_expect(!(e), 0) ? __assert_rtn(__func__, __ASSERT_FILE_NAME, __LINE__, #e) : (void)0)
                        ^
    (__builtin_expect(!(e), 0) ? __assert_rtn(__func__, __ASSERT_FILE_NAME, __LINE__, #e) : (void)0)
                        ^
In file included from ../fsevents.cc:6:
In file included from ../../../../nan/nan.h:2722:
../../../../nan/nan_object_wrap.h:127:26: In file included from ../fsevents.ccerror: :no member named 'IsNearDeath' in 'Nan::Persistent<v8::Object>'
6:
In file included from ../../../../nan/nan.h:2722:
../../../../nan/nan_object_wrap.h:    assert(wrap->handle_.IsNearDeath());127:26:
           ~~~~~~~~~~~~~ ^
 error: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/assert.h:99:25: no member named 'IsNearDeath' in 'Nan::Persistent<v8::Object>'
note: expanded from macro 'assert'
    assert(wrap->handle_.IsNearDeath());
           ~~~~~~~~~~~~~ ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/usr/include/assert.h:99:    (__builtin_expect(!(e), 0) ? __assert_rtn(__func__, __ASSERT_FILE_NAME, __LINE__, #e) : (void)0)
                        ^25: note: expanded from macro 'assert'

    (__builtin_expect(!(e), 0) ? __assert_rtn(__func__, __ASSERT_FILE_NAME, __LINE__, #e) : (void)0)
                        ^
../fsevents.cc:43:32: error: no template named 'Handle' in namespace 'v8'
../fsevents.cc:43:32: error: no template named 'Handle' in namespace 'v8'    static void Initialize(v8::Handle<v8::Object> exports);

                           ~~~~^
    static void Initialize(v8::Handle<v8::Object> exports);
                           ~~~~^
In file included from ../fsevents.cc:73:
../src/constants.cc:89:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
In file included from ../fsevents.cc:73:
../src/constants.cc:89:11  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagNone\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagNone));
          ^
: warning: /Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]:
3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagNone\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagNone));
          ^
328:29:/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h :note: 3498expanded from macro 'V8_DEPRECATED':
3: note: 'Set' has been explicitly marked deprecated here
  declarator __attribute__((deprecated(message)))
                            ^  V8_DEPRECATED(\"Use maybe version\",

  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:90:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagMustScanSubDirs\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagMustScanSubDirs));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.hIn file included from ../fsevents.cc::3498:373::
 note: ../src/constants.cc'Set' has been explicitly marked deprecated here:
90:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagMustScanSubDirs\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagMustScanSubDirs));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:91:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagUserDropped\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagUserDropped));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: In file included from ../fsevents.cc:73:
note: 'Set' has been explicitly marked deprecated here
../src/constants.cc:91:11  V8_DEPRECATED(\"Use maybe version\",
:  ^ 
warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h
:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagUserDropped\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagUserDropped));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:92:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagKernelDropped\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagKernelDropped));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
In file included from ../fsevents.cc:73  V8_DEPRECATED(\"Use maybe version\",:

../src/constants.cc  ^:
92:11/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:: 328warning: :'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]29
: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagKernelDropped\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagKernelDropped));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:93:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagEventIdsWrapped\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagEventIdsWrapped));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
In file included from ../fsevents.cc:73  V8_DEPRECATED(\"Use maybe version\",
  ^
:
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h../src/constants.cc:93:11:: 328:29:warning:  note: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]expanded from macro 'V8_DEPRECATED'

  declarator __attribute__((deprecated(message)))
                            ^
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagEventIdsWrapped\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagEventIdsWrapped));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:94:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagHistoryDone\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagHistoryDone));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:In file included from ../fsevents.cc:29: note: 73expanded from macro 'V8_DEPRECATED':

../src/constants.cc:94:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]  declarator __attribute__((deprecated(message)))

                            ^
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagHistoryDone\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagHistoryDone));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:95:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagRootChanged\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagRootChanged));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
In file included from ../fsevents.cc:73:
  declarator __attribute__((deprecated(message)))
                            ^
../src/constants.cc:95:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagRootChanged\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagRootChanged));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:96:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagMount\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagMount));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:96:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagMount\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagMount));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:97:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagUnmount\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagUnmount));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:97:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagUnmount\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagUnmount));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:98:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemCreated\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemCreated));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:98:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemCreated\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemCreated));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:99:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemRemoved\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemRemoved));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:99:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemRemoved\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemRemoved));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:100:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemInodeMetaMod\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemInodeMetaMod));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:100:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemInodeMetaMod\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemInodeMetaMod));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:101:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemRenamed\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemRenamed));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:101:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemRenamed\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemRenamed));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:102:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemModified\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemModified));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:102:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemModified\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemModified));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:103:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemFinderInfoMod\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemFinderInfoMod));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:103:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemFinderInfoMod\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemFinderInfoMod));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:104:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemChangeOwner\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemChangeOwner));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:104:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemChangeOwner\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemChangeOwner));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:105:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemXattrMod\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemXattrMod));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:105:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemXattrMod\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemXattrMod));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:106:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemIsFile\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemIsFile));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:106:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemIsFile\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemIsFile));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:107:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemIsDir\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemIsDir));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:107:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemIsDir\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemIsDir));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:108:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemIsSymlink\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemIsSymlink));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
In file included from ../fsevents.cc:73:
../src/constants.cc:108:11: warning: 'Set' is deprecated: Use maybe version [-Wdeprecated-declarations]
  object->Set(Nan::New<v8::String>(\"kFSEventStreamEventFlagItemIsSymlink\").ToLocalChecked(), Nan::New<v8::Integer>(kFSEventStreamEventFlagItemIsSymlink));
          ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8.h:3498:3: note: 'Set' has been explicitly marked deprecated here
  V8_DEPRECATED(\"Use maybe version\",
  ^
/Users/XXX/Library/Caches/node-gyp/12.22.3/include/node/v8config.h:328:29: note: expanded from macro 'V8_DEPRECATED'
  declarator __attribute__((deprecated(message)))
                            ^
../fsevents.cc:76:16: error: variable has incomplete type 'void'
void FSEvents::Initialize(v8::Handle<v8::Object> exports) {
               ^
../fsevents.cc:76:16: error: variable has incomplete type 'void'
void FSEvents::Initialize(v8::Handle<v8::Object> exports) {
               ^
../fsevents.cc:76:31: error: no member named 'Handle' in namespace 'v8'
../fsevents.cc:76:void FSEvents::Initialize(v8::Handle<v8::Object> exports) {
                          ~~~~^31: error: 
no member named 'Handle' in namespace 'v8'
void FSEvents::Initialize(v8::Handle<v8::Object> exports) {
                          ~~~~^
../fsevents.cc:76:48: error: expected '(' for function-style cast or type construction
../fsevents.cc:76:48: error: expected '(' for function-style cast or type construction
void FSEvents::Initialize(v8::Handle<v8::Object> exports) {
                                     ~~~~~~~~~~^
void FSEvents::Initialize(v8::Handle<v8::Object> exports) {
                                     ~~~~~~~~~~^
../fsevents.cc:76:50: error: use of undeclared identifier 'exports'../fsevents.cc:76:50: error: use of undeclared identifier 'exports'

void FSEvents::Initialize(v8::Handle<v8::Object> exports) {
                                                 ^
void FSEvents::Initialize(v8::Handle<v8::Object> exports) {
                                                 ^
../fsevents.cc:76:58: error: expected ';' after top level declarator
../fsevents.cc:76:58: error: expected ';' after top level declaratorvoid FSEvents::Initialize(v8::Handle<v8::Object> exports) {
                                                         ^

                                                         ;
void FSEvents::Initialize(v8::Handle<v8::Object> exports) {
                                                         ^
                                                         ;
22 warnings and 9 errors generated.
22 warnings and 9 errors generated.
make: *** [Release/obj.target/fse/fsevents.o] Error 1
make: *** [Release/obj.target/fse/fsevents.o] Error 1
gyp gypERR! build error 
 ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:276:12)
gyp ERR! System Darwin 21.4.0
gyp ERR! command \"/Users/XXX/.volta/tools/image/node/12.22.3/bin/node\" \"/Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js\" \"build\" \"--fallback-to-build\" \"--module=/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents/lib/binding/Release/node-v72-darwin-x64/fse.node\" \"--module_name=fse\" \"--module_path=/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents/lib/binding/Release/node-v72-darwin-x64\" \"--napi_version=8\" \"--node_abi_napi=napi\"
gyp ERR! cwd /Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents
gyp ERR! gypnode -v v12.22.3
 ERR!gyp  SystemERR! Darwin 21.4.0
 node-gyp -v v5.1.0
gyp ERR! not okgyp 
 ERR! command \"/Users/XXX/.volta/tools/image/node/12.22.3/bin/node\" \"/Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js\" \"build\" \"--fallback-to-build\" \"--module=/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents/lib/binding/Release/node-v72-darwin-x64/fse.node\" \"--module_name=fse\" \"--module_path=/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents/lib/binding/Release/node-v72-darwin-x64\" \"--napi_version=8\" \"--node_abi_napi=napi\"
gyp ERR! cwd /Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents
gyp ERR! node -v v12.22.3
gyp ERR! node-gyp -v v5.1.0
gyp ERR! not ok 
node-pre-gyp ERR! build error 
node-pre-gyp ERR! stack Error: Failed to execute '/Users/XXX/.volta/tools/image/node/12.22.3/bin/node /Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents/lib/binding/Release/node-v72-darwin-x64/fse.node --module_name=fse --module_path=/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents/lib/binding/Release/node-v72-darwin-x64 --napi_version=8 --node_abi_napi=napi' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/node-pre-gyp/lib/util/compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:314:20)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:1022:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:287:5)
node-pre-gyp ERR! System Darwin 21.4.0
node-pre-gyp ERR! command \"/Users/XXX/.volta/tools/image/node/12.22.3/bin/node\" \"/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/node-pre-gyp/bin/node-pre-gyp\" \"install\" \"--fallback-to-build\"
node-pre-gyp ERR! cwd /Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_modules/fsevents
node-pre-gyp ERR! node -v v12.22.3
node-pre-gyp ERR! node-pre-gyp -v v0.10.3
node-pre-gyp ERR! not ok 
Failed to execute '/Users/XXX/.volta/tools/image/node/12.22.3/bin/node /Users/XXX/.volta/tools/image/node/12.22.3/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js build --fallback-to-build --module=/Users/XXX/dev/simplabs/simplabs.github.io/node_modules/chokidar/node_mod
✨  Done in 83.75s.
  ```
</details>

--- a/README.md
+++ b/README.md
@@ -43,6 +43,16 @@ Chrome Inspector to do so.**
 - [CSS Blocks](https://css-blocks.com)
 - [Puppeteer](https://pptr.dev)
 
+## Troubleshooting
+
+ <details>
+  <summary><code>yarn start-dev</code> fails in my local machine due to too many open files to watch</summary>
+  
+  You need to increase the max open files limit in your Operating System: https://wilsonmar.github.io/maximum-limits/
+  
+  <a href="https://wilsonmar.github.io/maximum-limits/">More details here</a>
+</details>
+
 ## Copyright
 
 Copyright &copy; 2019 simplabs GmbH (https://simplabs.com), released under the

